docs/articles/getting-started.md

```diff
@@ -4,34 +4,33 @@ uid: getting-started
 
 # Getting Started
 
-## 1. How to install Colore
 [![NuGet version][ngverbadge]][ng]
 [![MyGet version][mgverbadge]][mg]
 
-The easiest way to include Colore into your project is to right click on your C# Project in the Visual Studio Solution Explorer and Choose "Manage NuGet Packages..."
+## 1. Installing Chroma SDK
+Razer Chroma SDK is a C++-tool which allows us to access Chroma devices. It is the backend (or native/unmanaged code) behind Colore. It is expected to be automatically installed with Synapse once you plug in a Chroma device, but in some cases that might not work (see [#263](https://github.com/chroma-sdk/Colore/issues/263#)). As a last resort, you can also find an SDK installer in their [Setting Up](https://developer.razer.com/works-with-chroma/setting-up/) guide.
 
-From there you can search online for Packages.
-The NuGet Package manager sadly looks way different depending on the Version of Visual Studio but it should be easy to find Colore.
-Just search for "Colore" and install it.
-After that you should see a Reference being added to "Colore".
+## 2. How to install Colore 
+The easiest way include Colore into your project is to right click on your C# Project in the Visual Studio Solution Explorer and Choose "Manage NuGet Packages..."
```

Comment:
Should be "The easiest way <ins>to</ins> include"

Reply:
Ok, next time I´ll be more careful before submitting a PR